helios-sample/src/main/kotlin/helios/sample/sample.kt

```diff
@@ -27,57 +28,67 @@ const val companyJsonString = """
   ]
 }"""
 
-fun main(args: Array<String>) {
+  @JvmStatic
+  fun main(args: Array<String>) {
 
-  val companyJson: Json = Json.parseUnsafe(companyJsonString)
+    val companyJson: Json = Json.parseUnsafe(companyJsonString)
```

Comment:
This should all stay 2 spaced

Reply:
It's 2 spaced but it's in a object now to be able to run the sample